### ISSUE TITLE
Loading Set of Enum is failing

### ISSUE THREAD

**Issue by geomagilles** [sksamuel/hoplite#181]
Failing Test added in https://github.com/sksamuel/hoplite/pull/180

**Comment by geomagilles** [comment]
Trying to load set of Enum triggers this error:
```
klass.java.enumConstants must not be null
java.lang.IllegalStateException: klass.java.enumConstants must not be null
```

**Comment by sksamuel** [comment]
Fixed, will release 1.3.10

**Comment by geomagilles** [comment]
> Fixed, will release 1.3.10

woohoo - it's probably the quickest open-source fix ever. Thx!
